### ISSUE TITLE
Translators' properties do not persist when there are multiple translators of the same device type.

### ISSUE THREAD

**Issue by m-chau** [openT2T/translators#142]
Currently when we enumerate platforms of the same provider, we create a translator for each platform, then use that translator to invoke the ‘get’ method to get the platform details. When I have two outlet (binary switch) devices and call the hub translator to enumerate them, the control ID of the first outlet translator gets overwritten by that of the second outlet device.

Below is the log I get by adding some debug output. You may see the source for SmartThings hub at my repository . 
```
SmartThings Binary Switch initializing...
    Initialize translator for control ID: 529fa2ef-668b-4e84-a101-c47968997d8a
SmartThings Binary Switch initializing...Done                                   => translator created for outlet 1

SmartThings Binary Switch initializing...
    Initialize translator for control ID: e7a7c755-647e-4b69-8853-d66af047be4c
SmartThings Binary Switch initializing...Done                                   => translator created for outlet 2

* HUB::Invoking GET method for control ID: 529fa2ef-668b-4e84-a101-c47968997d8a => before line 110 in SmartThings hub: OpenT2T.invokeMethodAsync(...) to get device details for outlet 1
** Translator::get for control ID: e7a7c755-647e-4b69-8853-d66af047be4c         => In the “get(expand, payload)” method of outlet 1’s translator

* HUB::Invoking GET method for control ID: e7a7c755-647e-4b69-8853-d66af047be4c
** Translator::get for control ID: e7a7c755-647e-4b69-8853-d66af047be4c
```
As shown in the log, the control ID of outlet 1’s translator is overwritten by that of outlet 2 even though I was invoking the method using the translator created for outlet 1.

**Comment by m-chau** [comment]
Fixes is included in PR #152 .

**Comment by danielrischar** [comment]
This should be fixed for Wink, SmartThings, Nest, and Insteon now.  Just need to get some more eyes on it.
